recipes/diligent-fx/all/test_package/test_package.cpp

```diff
@@ -1,4 +1,5 @@
 #include <DiligentFx/Components/interface/ShadowMapManager.hpp>
+#include <DiligentFX/GLTF_PBR_Renderer/interface/GLTF_PBR_Renderer.hpp>
```

Comment:
```suggestion
#include <DiligentFx/GLTF_PBR_Renderer/interface/GLTF_PBR_Renderer.hpp>
```

Reply:
Oh, I haven't noticed that. Thank you a lot!

Reply:
I will incorporate this small change to a bigger PR later